bigquery/storage/managedwriter/managed_stream.go

```diff
@@ -189,9 +189,13 @@ func (ms *ManagedStream) getStream(arc *storagepb.BigQueryWrite_AppendRowsClient
 	}
 
 	// Always return the retained ARC if the arg differs.
-	if arc != ms.arc {
+	if arc != ms.arc && !forceReconnect {
 		return ms.arc, ms.pending, nil
 	}
+	if arc != ms.arc && forceReconnect && ms.arc != nil {
+		// TODO: is closing send sufficient?
+		(*ms.arc).CloseSend()
```

Comment:
this smells fishy, or is it just me?

Reply:
It's a temporary issue until the backend allows schema change on an already open stream connection; I'm not enamored of it but this will get cleaned up.